### PR TITLE
Deleted units

### PR DESCRIPTION
**Pull request by SoAmano**

### CODE REVIEW

**Comment by SoAmano** [comment]
Deleted the units and amount. Let me know if you use them in any way in the recipe part.

File: app/src/main/java/com/example/cs65_final_project/FirebaseDatabaseHelper.java [hunk 1]
```diff
@@ -28,21 +28,19 @@
...
             public void onDataChange(@NonNull DataSnapshot snapshot) {
                 if (snapshot.exists()) { // Ingredient already exists in the database
-                    float existingAmount = snapshot.getValue(Float.class);
-                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue(existingAmount+amount);
+                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name);
                     Log.d("RecipeFinderDatabase", "Added to Existing Ingredient");
                     ((AppCompatActivity)(context)).finish();
                 } else { // Ingredient does not currently exist in the database
-                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue(amount);
+                    ref.child("users").child(auth.getUid()).child("fridge").child(aisle).child(name).setValue("0");
```
Comment (ammarmazen) [inline app/src/main/java/com/example/cs65_final_project/FirebaseDatabaseHelper.java:43]: Can we just get rid of the value here?

File: app/src/main/java/com/example/cs65_final_project/Ingredient.java [hunk 2]
```diff
@@ -17,6 +17,11 @@ public Ingredient(String name, float amount, @Nullable String aisle) {
         this.aisle = aisleClassifier(aisle);
     }
 
+    public Ingredient(String name, String aisle){
```
Comment (ammarmazen) [inline app/src/main/java/com/example/cs65_final_project/Ingredient.java:20]: Should we just remove `float amount` from the above constructor?

**Review by ammarmazen — APPROVED** [review]

File: app/src/main/java/com/example/cs65_final_project/FirebaseDatabaseHelper.java (hunk 1 above)
Comment (SoAmano) [reply app/src/main/java/com/example/cs65_final_project/FirebaseDatabaseHelper.java:43]: Thought I could make the amount an optional part, and just decided to put 0 as a placeholder

File: app/src/main/java/com/example/cs65_final_project/Ingredient.java (hunk 2 above)
Comment (SoAmano) [reply app/src/main/java/com/example/cs65_final_project/Ingredient.java:20]: Same thing here, just an optional constructor,

**Pull request merged**